yamlgen/deploy/bottlerocket-update-operator.yaml

```diff
@@ -203,7 +217,7 @@ spec:
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
-  name: brupop-apiserver
+  name: brupop-apiserver-cerfiticate
```

Comment:
Incorrect name still remain in Yaml file, maybe you need run `cargo build` to refresh the yaml file content. : )